modules/core/locales/en.json

```diff
@@ -245,6 +245,7 @@
         "Title": "Account"
       }
     },
+    "Avatar": "Browse filesystem",
```

Comment:
This should probably be at the root, not inside `Account` object, and it's a noun describing an action, making it inconsistent with others. Maybe `BrowseFilesystem` would be better

Reply:
@diyor28

Reply:
I'll rename it. The reason it's part of Account object is because it's a label passed from Account form to component